### PR TITLE
SAAS-99 Tests for STT in PMM Settings

### PR DESCRIPTION
**Pull request by yurkovychv**
covered test cases : 
- PMM-T253
- PMM-T254

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/grafana-dashboards?pullRequest=696) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/grafana-dashboards?pullRequest=696) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: puneet0191<br/>:x: yurkovychv<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/grafana-dashboards?pullRequest=696) it.</sub>

**Comment by puneet0191** [comment]
@yurkovychv sign CLA

File: pmm-app/tests/pages/pmmSettingsPage.js [hunk 1]
```diff
@@ -286,4 +297,31 @@ module.exports = {
     }
     I.see(ruleName);
   },
+
+  async verifyTooltip(tooltipObj) {
+    I.see(tooltipObj.text, this.fields.tooltipLocator);
+    I.seeAttributesOnElements(`${this.fields.tooltipLocator} > div > a`, {href: tooltipObj.link});
```
Comment (puneet0191) [inline pmm-app/tests/pages/pmmSettingsPage.js:303]: This locator can be improved, no other way to handle it?

File: pmm-app/tests/pages/pmmSettingsPage.js [hunk 2]
```diff
@@ -82,14 +91,16 @@ module.exports = {
     popUpTitle: "//div[@class='alert-title']",
     validationMessage: "//span[@class='error-message']",
     selectedResolution: "//span[@class='ant-slider-mark-text ant-slider-mark-text-active']",
+    sttSwitchSelector: "//strong[text()='Security Threat Tool']/parent::div/following-sibling::div/button",
+    telemetrySwitchSelector: "//strong[text()='Telemetry']/parent::div/following-sibling::div/button",
+    sttLabelTooltipLocator: "//strong[text()='Security Threat Tool']/following-sibling::span/i",
+    tooltipLocator: ".ant-tooltip-inner"
```
Comment (lunaticusgreen) [inline pmm-app/tests/pages/pmmSettingsPage.js:97]: I'd propose to use similar naming approach (Selector), and left locators for function that returns selectors

Comment (yurkovychv) [reply pmm-app/tests/pages/pmmSettingsPage.js]: yes, sorry missed that one

File: pmm-app/tests/pages/pmmSettingsPage.js (hunk 1 above)
Comment (yurkovychv) [reply pmm-app/tests/pages/pmmSettingsPage.js]: do you think we need to do such changes? the method is pretty simple and reusable now

**Pull request merged**